drivers/dac/ad9152/ad9152.h

```diff
@@ -86,14 +88,14 @@
 #define REG_DACGAIN2_1                           0x044 /* MSBs of Full Scale Adjust DAC */
```

Comment:
since you're at it, remove registers 0x44 - 0x46, they don't appear in the current datasheet